src/main/java/org/kainos/ea/DropwizardWebServiceConfiguration.java

```diff
@@ -1,9 +1,8 @@
 package org.kainos.ea;
 
 import io.dropwizard.Configuration;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import org.hibernate.validator.constraints.*;
-import javax.validation.constraints.*;
+//import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
Please delete lines instead of commeting them out

Reply:
change made